internal/warehouse/github_actions.go

```diff
@@ -213,11 +223,21 @@ func (w *warehouse) handleWorkflowRunsJobs(ctx context.Context, owner, repo stri
 	}
 
 	for _, workflowRun := range workflowRunsPage {
+
+		if workflowRun.ID == nil {
+			w.logger.Debug().Msg("workflow run with empty ID")
+			break
+		}
 		// we get a page of 30 workflow run jobs until next page is 0
 		for {
-			w.logger.Debug().Str("workflow-run", *workflowRun.Name).Str("ID", fmt.Sprintf("%d", *workflowRun.ID)).Msg("getting jobs of")
+			if workflowRun.Name != nil {
+				w.logger.Debug().Str("workflow-run", *workflowRun.Name).Str("ID", fmt.Sprintf("%d", *workflowRun.ID)).Msg("getting jobs of")
+			}
+
 			if workflowRunJobsPage, resp, err = getWorkflowJobs(ctx, w.githubClient, owner, repo, *workflowRun.ID, opt); err != nil {
-				w.logger.Warn().Str("workflow-run", *workflowRun.Name).Str("ID", fmt.Sprintf("%d", *workflowRun.ID)).AnErr("Error", err).Msg("error occurred")
+				if workflowRun.Name != nil {
+					w.logger.Warn().Str("workflow-run", *workflowRun.Name).Str("ID", fmt.Sprintf("%d", *workflowRun.ID)).AnErr("Error", err).Msg("error occurred")
+				}
```

Comment:
```suggestion
				w.logger.Warn().Str("workflow-run", *workflowRun.Name).Str("ID", fmt.Sprintf("%d", *workflowRun.ID)).AnErr("error", err).Msg("error occurred")
```

Sme here - don't we do the check above?